src/Symfony/Bridge/Twig/Tests/Extension/AbstractBootstrap3HorizontalLayoutTest.php

```diff
@@ -21,7 +21,7 @@ public function testLabelOnForm()
         $html = $this->renderLabel($view);
 
         $this->assertMatchesXpath($html,
```

Comment:
To keep this change this should be full multilines args:
```suggestion
        $this->assertMatchesXpath(
            $html,
            '...
```
or 
```suggestion
        $this->assertMatchesXpath($html, '
...
```